distribution/docs/src/main/resources/content/_running/ingesting.adoc

```diff
@@ -10,6 +10,12 @@ Ingesting is the process of getting metacard(s) into the Catalog Framework.
 Ingested files are "transformed" into a neutral format that can be searched against as well as migrated to other formats and systems.
 There are multiple methods available for ingesting files into the ${branding}.
 
+.Guest Claims Attributes and Ingest
+[NOTE]
+====
+Ensure that appropriate <<_configuring_guest_interceptor,Guest Claims>> are configured to allow ingest and query of the data to be ingested.
```

Comment:
`to allow guest to ingest data and query the catalog.`?

Reply:
fixed.